db/fixtures/miq_policies.yml

```diff
@@ -0,0 +1,29 @@
+---
+- :name: analyze_new_images
+  :description: Analyze all new images
```

Comment:
Based on my reasoning [here](https://github.com/ManageIQ/manageiq/pull/7966/files#r59782489), this should probably be OpenSCAP specific in the description and name as well.  Mabye something like "Analyse all new containers images for known vulnerabilities with OpenSCAP ruleset"